### ISSUE TITLE
Codepage detection in MacPreview does not work

### ISSUE THREAD

**Issue by lprot** [doublecmd/doublecmd#1726]
### Version

Double Commander
Version: 1.2.0 alpha
Revision: 11376
Commit: e41090916
Build date: 2024/06/20
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 12.7.5

### Description

[CodingLog.csv](https://github.com/user-attachments/files/15917406/CodingLog.csv)
[AdpLog.CSV](https://github.com/user-attachments/files/15917422/AdpLog.CSV)
![image](https://github.com/doublecmd/doublecmd/assets/2608545/6aa4d8f6-1a9c-4620-8f39-eb6282d6aea0)

Ref. https://github.com/doublecmd/doublecmd/issues/1724

**Comment by alexx2000** [comment]
I guess it is unfixable.

**Comment by lprot** [comment]
Can MacPreview be disabled for CSVs then?

**Comment by alexx2000** [comment]
> Can MacPreview be disabled for CSVs then?

Yes, you can disable it in the [plugin settings](https://doublecmd.github.io/doc/en/configuration.html#ConfigPlugins), by adding detect string:
`EXT!="CSV"`

> This strange plugin does not even support codepages:

It is not strange. WLX-API does not have such capability in general (only ANSI and OEM). Anyway as I see, Quick Look doesn't have a such function.

**Comment by lprot** [comment]
> Yes, you can disable it in the [plugin settings](https://doublecmd.github.io/doc/en/configuration.html#ConfigPlugins), by adding detect string: `EXT!="CSV"`

Thanks, this helped.
